velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -515,6 +515,40 @@ struct DateFromUnixDateFunction {
   }
 };
 
+/// Truncates a timestamp to a specified time unit.
```

Comment:
Perhaps document the behaviour for invalid format, and microseconds and abbreviated unit string is allowed.